examples/03_annotation/04_plot_gclda.py

```diff
@@ -113,9 +113,11 @@
 
 ###############################################################################
 # First we'll make an ROI
+hdr = nib.Nifti1Header()  # Passing an int64 array to Nifti1Image requires a header
+hdr.set_data_dtype(np.int64)
 arr = np.zeros(dset.masker.mask_img.shape, int)
```

Comment:
Alternative, you could use `np.int32` as the dtype here. Then you wouldn't need to create and pass in the header.